src/pymor/discretizers/builtin/cg.py

```diff
@@ -1013,6 +1013,11 @@ def discretize_stationary_cg(analytical_problem, diameter=None, domain_discretiz
         Must be provided if `grid` is specified.
     preassemble
         If `True`, preassemble all operators in the resulting |Model|.
+    energy_product
+        If not `None`, a |Parameter| `mu` for which to assemble the symmetric part of the |Operator| of the resulting
+        |Model| `fom`, such that `fom.operator.assemble(mu) == fom.products['energy']` (apart from the fact that the
+        latter includes Dirichlet unit rows and columns, while the former only includes Dirichlet rows), if
+        `fom.operator` is already symmetric. Only available if `preassemble == False`.
```

Comment:
I guess you mean 'if `preassemble is True`'. You can use `FixedParameterOperator` to make this work also when `preassemble is False`.